### DISCUSSION TITLE
Unable to use threadcount property

### DISCUSSION THREAD

**Discussion by Amishk2** [quartznet/quartznet#1760] [Q&A]
We were using quartz.threadPool.threadCount to define the threadcount in earlier version. 
and used below code  
            var s= new StdSchedulerFactory(properties);
            this.Scheduler = s.GetScheduler().Result;

After upgrading, unable to find the use of thread count for increading/decreasing number of thread. in the source code also unable to find it. Please suggest how the value of quartz.threadPool.threadCount get implected. 

**Version used** 3.4.0.0

With which version did you encounter this behavior.

**To Reproduce**

Ideally code samples or event unit test.

**Expected behavior** T

**Additional context**

**Comment by lahma** [comment]
It should work the same way as before if you are using the property `quartz.threadPool.threadCount`.

**Reply by Amishk2** [reply]
Can you please share the details where this property has been used. I have verified in the source code, and unable to find it. Its not here in StdSchedulerFactory.cs.

**Reply by lahma** [reply]
Searching the repo gives you these results: https://github.com/quartznet/quartznet/search?q=threadCount
